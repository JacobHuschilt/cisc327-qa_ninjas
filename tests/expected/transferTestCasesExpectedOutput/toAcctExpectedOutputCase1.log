OK. Second account verified in agent mode.