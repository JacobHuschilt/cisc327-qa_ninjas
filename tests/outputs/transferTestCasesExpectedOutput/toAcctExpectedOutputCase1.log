second account verified in agent mode