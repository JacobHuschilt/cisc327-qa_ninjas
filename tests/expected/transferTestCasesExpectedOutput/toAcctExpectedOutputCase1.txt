XFR 3487200 3426571 3500
EOS 0000000 000 0000000 ***